charts/streams-app/values.yaml

```diff
@@ -50,6 +50,21 @@ jmx:
   metricRules:
     - ".*"
 
+autoscaling:
+  enabled: false
+  # consumergroup: foo
+  lagThreshold: "1000"
+  pollingInterval: 30
+  cooldownPeriod: 300
+  offsetResetPolicy: earliest
+  minReplicas: 0
+  maxReplicas: 50
```

Comment:
What do you think about also commenting this out? I think max replicas should not have a default value to no unintentionally have 50 replicas by accident, causing problems in smaller clusters.

Reply:
Good point